Automattic/jetpack
# Admin Page: fix search results to only display Earn when necessary

Follow-up from #31696

## Proposed changes:

We need to introduce a fake "earn" module to be able to display the Earn section elements in search results only when necessary.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* no

## Testing instructions:

* Go to Jetpack > Settings
* Search for "widgets"
* You should not see any "Earn" elements.
* Search for "stripe"
* You should see Earn elements.
